### PR TITLE
[13.0][MIG] hr_attendance_geolocation

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@AdriaGForgeFlow Please, can you fix/adapt the javascript file?

**Comment by AdriaGForgeFlow** [comment]
> @AdriaGForgeFlow Please, can you fix/adapt the javascript file?

Yes. OMW

**Comment by pedrobaeza** [comment]
Can you please fix Travis?

File: hr_attendance_geolocation/models/hr_employee.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    _inherit = "hr.employee"
+
+    def attendance_manual(self, next_action, entered_pin=False, location=False):
+        res = super(
+            HrEmployee, self.with_context(attendance_location=location)
+        ).attendance_manual(next_action, entered_pin)
+        return res
+
+    def attendance_action_change(self):
+        res = super().attendance_action_change()
```
Comment (victoralmau) [inline hr_attendance_geolocation/models/hr_employee.py]:
```suggestion
    def _attendance_action_change(self):
        res = super()._attendance_action_change()
```

**Review by victoralmau — CHANGES_REQUESTED** [review]
These changes will solve Travis error

**Comment by MiquelRForgeFlow** [comment]
Ready :green_apple:

**Review by pedrobaeza — APPROVED** [review]

**Review by victoralmau — APPROVED** [review]

**Pull request merged**